components/pagerduty/actions/find-oncall-user/find-oncall-user.mjs

```diff
@@ -0,0 +1,47 @@
+import pagerduty from "../../pagerduty.app.mjs";
+
+export default {
+  key: "pagerduty-find-oncall-user",
+  name: "Find Oncall User",
+  description: "Find the user on call for a specific schedule. [See the docs here](https://developer.pagerduty.com/api-reference/b3A6Mjc0ODE0MA-create-an-incident)",
```

Comment:
Is the doc link actually this one?
https://developer.pagerduty.com/api-reference/b3A6Mjc0ODE5MA-list-users-on-call